### ISSUE TITLE
add support for other ensemble models

### ISSUE THREAD

**Issue by nidhaloff** [nidhaloff/igel#15]
### Description

Ensemble models like adaboost and gradient boosting should be supported. 
https://scikit-learn.org/stable/modules/classes.html?highlight=neural%20network#module-sklearn.neural_network

**Comment by nidhaloff** [comment]
done in v0.2.1
